### DISCUSSION TITLE
Location offset

### DISCUSSION THREAD

**Discussion by 333bkz** [mapsforge/vtm#1108] [Q&A] [answered]
I update the current latitude and longitude to the MapPosition so that the current location is always displayed at the center of the map. 
But now I want to display the current location in the lower half of the map.
How should I do this?

**Comment by 333bkz** [comment]
<img width="380" alt="image" src="https://github.com/mapsforge/vtm/assets/38521620/c3f88e6a-7ace-4bfa-b3b9-d01bc94884e9">
I want to achieve this effect so that users can better see the route ahead

**Comment by 333bkz** [comment]
I will try to use `map().viewport().setMapViewCenterY(float)`

**Comment by devemux86** [comment, marked as answer]
> I will try to use map().viewport().setMapViewCenterY(float)

:+1:

**Reply by 333bkz** [reply]
<img width="469" alt="image" src="https://github.com/mapsforge/vtm/assets/38521620/91bc53db-2159-4367-9149-d09f3b0179b8">
map.viewport().mapViewCenterY = .6f
🍻
